gaj23/face-it
# Gaj/testing/search bar

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

This branch includes a bulk of the happy path testing, as well as some minor bug fixes. This branch is in preparation for deployment. 


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.



### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see the cypress testing folder for code related to UX/UI testing. 

Please see Type for logic to remove underscore.

Please see Footer to view imgs added rather than external links.

### How should this be tested?
> What steps need to be taken to confirm functionality?
Please ensure that cypress is downloaded locally. run `npm run cypress` to check all tests are passing. 


### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
